### ISSUE TITLE
Investigate removing (more) permissions from the dataset permissions table that are not relevant to datasets

### ISSUE THREAD

**Issue by jggautier** [IQSS/dataverse#6067]
The dataset permissions page lists roles that users might find confusing because they don't apply to the dataset in question. For example, in the screenshot of the dataset permissions page below, ":authenticatedusers" are dataset creators. That role was set at the dataverse level, for the dataverse of this dataset, so that anyone with an account can create datasets in the "Julian Gautier, Dataverse". Does it make sense to include this information on the **dataset** permissions page?

![Screen Shot 2019-08-01 at 3 03 04 PM](https://user-images.githubusercontent.com/18374574/62320140-82c69c80-b46d-11e9-997b-638165a4ff14.png)

GitHub suggested that this resolved issue is relevant: https://github.com/IQSS/dataverse/issues/1370

**Comment by jggautier** [comment]
In the Dataverse Users Community, there's [a conversation from 2016 related to this issue](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/dataverse-community/XjodQ6bQjFg/2rrRcAqECAAJ).

**Comment by sbarbosadataverse** [comment]
@jggautier who can see this permissions page? Only you as an admin/curator, correct? I want to be sure " contributors" can't see this page...as they shouldn't be able to see it.

**Comment by scolapasta** [comment]
Yes, only those with ManageDatasetPermissions (Admin and Curator) can see this page.

**Comment by sbarbosadataverse** [comment]
In that case, yes this is often confusing because I have to go back myself and check who was assigned at the Dataverse level, and is showing up at the dataset level, etc...

**Comment by scolapasta** [comment]
Well, it's not so much about what level you were assigned at, but what the role is. While there's no reason to show roles that only have Dataverse level permissions at the Dataset level, you *would* want to show if, for example, someone is assigned Contributor, at the Dataverse level. That is a valid use case (I want this person to be able to contribute to any Dataset in this Dataverse). As another example, the Curator role is often assigned at the Dataverse level in order to curate any Datasets.

**Comment by sbarbosadataverse** [comment]
"if, for example, someone is assigned Contributor, at the Dataverse level. That is a valid use case (I want this person to be able to contribute to any Dataset in this Dataverse)."---- 

This not being the same as a parent "dataverse" permissions being set to allow depositors as "contributors" where they can deposit a dataset, and only edit their own dataset, not datasets deposited by other ""contributors?"--correct? We can discuss off this ticket.

**Comment by scolapasta** [comment]
That is correct, it is not the same.

That setting is for what role is given to the person who deposits the dataset. What I'm describing is a role that cascaded down. From my previous post the curator might be more straightforward to follow. I assign the Curator role at the dataverse level, so that that person then has the Curator role for any and all datasets in that dataverse (by virtue of the cascading).

**Comment by scolapasta** [comment]
Yes, it is the same issue, so I will close this one in favor of that one (which has the attached PR to it). I'm happy it's finally getting addressed. :)
